main.md

```diff
@@ -245,7 +242,7 @@ Clients MAY use the `request` parameter as defined in JAR [@!I-D.ietf-oauth-jwsr
 
 Clients MUST NOT combine other authorization request parameters with the `request` parameter at the pushed authorization request endpoint other than the `client_id` parameter which may be a part of the client authentication mechanism.
 
-The following is an example of a request using a signed request object. The client is authenticated using its client secret `BASIC` authorization:
+The following is an example of a request using a signed request object. The client is authenticated using its client secret in Basic Authentication:
```

Comment:
"using basic authentication" or "in Basic authorization header"?

Reply:
"Basic authorization header" sounds strange. Among the two, "using basic authentication" sounds better.

Reply:
the suggestion @jricher made below sounds right to me